.gitignore

```diff
@@ -4,6 +4,9 @@ dist-newstyle/
 *.hi
 *.o
 
+# Cabal Project user-local part
+cabal.project.local
```

Comment:
I agree that GitHub should ignore this file. I've now pushed something separately to the `master` branch, so this should fall away.